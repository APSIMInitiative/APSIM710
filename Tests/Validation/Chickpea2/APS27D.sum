     ###     ######     #####   #   #     #   
    #   #    #     #   #        #   ##   ##   
   #     #   #     #   #        #   ##   ##   
   #######   ######     #####   #   # # # #   
   #     #   #              #   #   #  #  #   
   #     #   #         #####    #   #  #  #   
                                              
                                              
 The Agricultural Production Systems Simulator
             Copyright(c) APSRU               

Version                = 7.4
Title                  = APS27D
   Component                        "clock" = %apsim%\Model\Clock.dll
   Component                          "met" = %apsim%\Model\Input.dll
Paddock:
   Component                   "outputfile" = %apsim%\Model\Report.dll
   Component                        "accum" = %apsim%\Model\Accum.dll
   Component                   "Operations" = %apsim%\Model\Operations.dll
   Component                   "fertiliser" = %apsim%\Model\Fertiliser.dll
   Component                   "irrigation" = %apsim%\Model\Irrigation.dll
   Component                "output titles" = %apsim%\Model\Manager.dll
   Component              "Harvesting rule" = %apsim%\Model\Manager.dll
   Component                     "Micromet" = %apsim%\Model\MicroMet.dll
   Component                   "soil Water" = %apsim%\Model\SoilWat.dll
   Component                    "SurfaceOM" = %apsim%\Model\SurfaceOM.dll
   Component                "soil Nitrogen" = %apsim%\Model\SoilN.dll
   Component                    "chickpea2" = %apsim%\Model\Plant2.dll

------- clock Initialisation --------------------------------------------------
     Sequencer phases:
        prepare
        init_step
        do_pasture_water
        process
        do_pasture_growth
        do_stock
        post
        end_step
     Simulation start date = 14/06/1995
     Simulation end date   = 26/11/1995
     Time step =           = 1440 (mins)

------- met Initialisation ----------------------------------------------------
     Sparse data is not allowed
     INPUT File name: met\cooper.met

------- outputfile Initialisation ---------------------------------------------
     Output frequency:
        end_day
     Output variables:
        day
        Expcode as Title
        Site
        Season
        Treatment
        dd/mm/yyyy as Date
        DaysAfterSowing
        TTAfterSowing
        sw_dep
        AboveGroundWt
        RootDepth
        StageCodeValue
        LeafPhotosynthesisFtValue
        LeafPhotosynthesisFvpdValue
        LeafFw
        LeafLAI
        swTotal
        ll15_dep
        GrainLiveWt

     Output file = APS27D.out
     Format = normal

------- accum Initialisation --------------------------------------------------
     Initialising

------- Operations Initialisation ---------------------------------------------
     Operations Schedule
     ===================
     15/06/1995  Prepare      chickpea2 sow population= 39.5, depth= 50, cultivar = amethyst, maxcover=1.0, budNumber=1
      22/6/1995  Prepare      irrigation apply amount = 3
     22/06/1995  Prepare      irrigation apply amount = 5.25
      23/6/1995  Prepare      irrigation apply amount = 3
       5/7/1995  Prepare      irrigation apply amount = 7
      13/7/1995  Prepare      irrigation apply amount = 24

------- irrigation Initialisation ---------------------------------------------
      Initialising
     
        - Reading Parameters
                      Irrigation parameters
          -----------------------------------------------
           Automatic Irrigation Application (Disabled)
           critical fraction of available soil water =  0.00
           depth for calculating available soil water =       0.00
           Irrigation Allocation Budget (Disabled)
          -----------------------------------------------

------- output titles Initialisation ------------------------------------------
     Manager rules:
     
     SECTION:- init
     
     SECTION:- start_of_day
     expcode='aps27d'
          site='aps27w '
          season='-y1995'
          treatment='amethyst'
     
     SECTION:- end_of_day
     swtotal = sw_dep()-sw_dep(1)
     END of rules
     

------- Harvesting rule Initialisation ----------------------------------------
     Manager rules:
     
     SECTION:- end_of_day
                if (chickpea2.stagename = 'harvest_ripe' or chickpea2.plant_status = 'dead') then
       chickpea2  harvest
       chickpea2  end_crop
                endif
     END of rules
     

------- Micromet Initialisation -----------------------------------------------
     Initialising
     
        - Reading Constants
     
        - Reading Parameters

------- soil Water Initialisation ---------------------------------------------
     
         - Reading constants
     
        - Reading Soil Property Parameters
     
        - Reading Soil Profile Parameters
     Initial soilwater distributed using "sw" parameter.
     
     
     
                      Soil Profile Properties
        ---------------------------------------------------------------------
              Depth  Air_Dry  LL15   Dul    Sat     Sw     BD   Runoff  SWCON
                mm     mm/mm  mm/mm  mm/mm  mm/mm  mm/mm  g/cc    wf
        ---------------------------------------------------------------------
            0.-  150.  0.050  0.250  0.350  0.434  0.340  1.380  0.762  0.400
          150.-  300.  0.190  0.250  0.350  0.431  0.346  1.380  0.190  0.400
          300.-  600.  0.200  0.200  0.310  0.490  0.309  1.320  0.048  0.400
          600.-  900.  0.220  0.230  0.325  0.448  0.324  1.460  0.000  0.400
          900.- 1200.  0.190  0.210  0.300  0.430  0.302  1.510  0.000  0.400
         1200.- 1500.  0.170  0.220  0.290  0.426  0.290  1.490  0.000  0.400
         1500.- 1800.  0.170  0.220  0.280  0.427  0.278  1.470  0.000  0.400
        ---------------------------------------------------------------------
     
     
     
                  Soil Water Holding Capacity
          ---------------------------------------------------------
              Depth    Unavailable Available  Max Avail.  Drainable
                          (LL15)   (SW-LL15)  (DUL-LL15)  (SAT-DUL)
                            mm        mm          mm         mm
          ---------------------------------------------------------
            0.-  150.      37.50      13.50      15.00      12.60
          150.-  300.      37.50      14.40      15.00      12.15
          300.-  600.      60.00      32.70      33.00      54.00
          600.-  900.      69.00      28.20      28.50      36.90
          900.- 1200.      63.00      27.60      27.00      39.00
         1200.- 1500.      66.00      21.00      21.00      40.80
         1500.- 1800.      66.00      17.40      18.00      44.10
          ---------------------------------------------------------
               Totals     399.00     154.80     157.50     239.55
          ---------------------------------------------------------
     
     
     
     
     
     
                  Initial Soil Parameters
          ---------------------------------------------------------
                 Insoil        Salb     Dif_Con   Dif_Slope
          ---------------------------------------------------------
                   0.00        0.13       40.00       16.00
          ---------------------------------------------------------
     
     
     
                  Runoff is predicted using scs curve number:
                Cn2  Cn_Red  Cn_Cov   H_Eff_Depth
                                           mm
          ---------------------------------------------------------
              72.00   20.00    0.80  450.00
          ---------------------------------------------------------
     
     
     
           Using Ritchie evaporation model
            Cuml evap (U):            6.00 (mm^0.5)
            CONA:                     2.00 ()
            Eo from priestly-taylor

------- SurfaceOM Initialisation ----------------------------------------------
     
         - Reading constants
     
         - Reading parameters
     
     
     
                         Initial Surface Organic Matter Data
         ----------------------------------------------------------------------
            Name   Type        Dry matter   C        N        P    Cover  Standing_fr
                                (kg/ha)  (kg/ha)  (kg/ha)  (kg/ha) (0-1)     (0-1)
         ----------------------------------------------------------------------
          unknown   wheat            0.0     0.0     0.0     0.0   0.000     0.0
         ----------------------------------------------------------------------
     
                      Effective Cover from Surface Materials =   0.0
     

------- soil Nitrogen Initialisation ------------------------------------------
     
        - Reading Parameters
     
        - Reading Constants
     
     
     
           TAV and AMP supplied externally
     
     
     
                      Soil Profile Properties
          ------------------------------------------------
           Layer    pH    OC     NO3     NH4    Urea
                         (%) (kg/ha) (kg/ha) (kg/ha)
          ------------------------------------------------
             1     7.50  1.20  124.20    0.00    0.00
             2     8.00  0.80   82.80    0.00    0.00
             3     8.00  0.50  158.40    0.00    0.00
             4     8.00  0.30   43.80    0.00    0.00
             5     8.00  0.20    4.53    0.00    0.00
             6     8.00  0.10    0.89    0.00    0.00
             7     8.00  0.10    0.44    0.00    0.00
          ------------------------------------------------
           Totals              415.07    0.00    0.00
          ------------------------------------------------
     
     
     
     
     
     
                  Initial Soil Organic Matter Status
          ---------------------------------------------------------
           Layer      Hum-C   Hum-N  Biom-C  Biom-N   FOM-C   FOM-N
                    (kg/ha) (kg/ha) (kg/ha) (kg/ha) (kg/ha) (kg/ha)
          ---------------------------------------------------------
             1      24528.3  1691.6   311.7    39.0    69.4     0.9
             2      16437.6  1133.6   122.4    15.3    54.1     0.7
             3      19719.6  1360.0    80.4    10.0    32.8     0.4
             4      13119.2   904.8    20.8     2.6    19.9     0.2
             5       9042.1   623.6    17.9     2.2    12.1     0.2
             6       4469.6   308.2     0.4     0.1     7.3     0.1
             7       4409.6   304.1     0.4     0.1     4.4     0.1
          ---------------------------------------------------------
           Totals   91725.9  6325.9   554.1    69.3   200.0     2.5
          ---------------------------------------------------------
------- Start of simulation  --------------------------------------------------
     Manager creating a new local string variable : expcode = aps27d
     Manager creating a new local string variable : site = aps27w
     Manager creating a new local string variable : season = -y1995
     Manager creating a new local string variable : treatment = amethyst
     Manager creating a new local real variable : swtotal = 502.78793
15 June 1995(Day of year=166), Operations: 
      Sending sow message to chickpea2
16/06/1995 - Germination
                            LAI = 0.00 (m^2/m^2)
           Above Ground Biomass = 0.00 (g/m^2)
22 June 1995(Day of year=173), Operations: 
      Sending apply message to irrigation
      Sending apply message to irrigation
23 June 1995(Day of year=174), Operations: 
      Sending apply message to irrigation
26/06/1995 - Emergence
                            LAI = 0.00 (m^2/m^2)
           Above Ground Biomass = 0.00 (g/m^2)
5 July 1995(Day of year=186), Operations: 
      Sending apply message to irrigation
13 July 1995(Day of year=194), Operations: 
      Sending apply message to irrigation
2/08/1995 - EndJuvenile
                            LAI = 0.39 (m^2/m^2)
           Above Ground Biomass = 18.05 (g/m^2)
25/08/1995 - FloralInitiation
                            LAI = 2.38 (m^2/m^2)
           Above Ground Biomass = 114.75 (g/m^2)
26/08/1995 - Flowering
                            LAI = 2.50 (m^2/m^2)
           Above Ground Biomass = 121.47 (g/m^2)
20/09/1995 - StartPodGrowth
                            LAI = 3.85 (m^2/m^2)
           Above Ground Biomass = 287.33 (g/m^2)
25/10/1995 - EndGrainFill
                            LAI = 0.38 (m^2/m^2)
           Above Ground Biomass = 434.03 (g/m^2)
27/10/1995 - Maturity
                            LAI = 0.25 (m^2/m^2)
           Above Ground Biomass = 436.16 (g/m^2)
28/10/1995 - HarvestRipe
                            LAI = 0.20 (m^2/m^2)
           Above Ground Biomass = 438.58 (g/m^2)
26 November 1995(Day of year=330), clock: 
     Simulation is terminating due to end criteria being met.
