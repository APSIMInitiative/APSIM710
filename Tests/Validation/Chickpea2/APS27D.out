ApsimVersion = 7.6
Title = APS27D
            day          Title           Site         Season      Treatment           Date DaysAfterSowing  TTAfterSowing      sw_dep(1)      sw_dep(2)      sw_dep(3)      sw_dep(4)      sw_dep(5)      sw_dep(6)      sw_dep(7)  AboveGroundWt      RootDepth StageCodeValue LeafPhotosynthesisFtValue LeafPhotosynthesisFvpdValue         LeafFw        LeafLAI        swTotal    ll15_dep(1)    ll15_dep(2)    ll15_dep(3)    ll15_dep(4)    ll15_dep(5)    ll15_dep(6)    ll15_dep(7)    GrainLiveWt
          (day)             ()             ()             ()             ()   (dd/mm/yyyy)              ()             ()           (mm)           (mm)           (mm)           (mm)           (mm)           (mm)           (mm)             ()             ()             ()                        ()                          ()             ()             ()             ()           (mm)           (mm)           (mm)           (mm)           (mm)           (mm)           (mm)             ()
            165         aps27d         aps27w         -y1995       amethyst     14/06/1995               ?              ?         49.834         51.850         92.744         97.194         90.360         87.137         83.503              ?              ?              ?                         ?                           ?              ?              ?        502.788         37.500         37.500         60.000         69.000         63.000         66.000         66.000              ?
